$NetBSD: patch-src_zmc_cpp,v 1.4 2022/09/27 01:20:39 gdt Exp $

Use a more appropriate initialisation for a long, and use the definition out
of <limits.h>, for portability.

--- src/zmc.cpp.orig	2019-02-22 15:38:47.000000000 +0000
+++ src/zmc.cpp
@@ -55,7 +55,7 @@ possible, this should run at more or les
 
 #include <getopt.h>
 #include <signal.h>
-#if defined(__FreeBSD__)
+#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__APPLE__)
 #include <limits.h>
 #else
 #include <values.h>
@@ -280,7 +280,7 @@ int main( int argc, char *argv[] )
     sigprocmask( SIG_BLOCK, &block_set, 0 );
     for ( int i = 0; i < n_monitors; i++ )
     {
-      long min_delay = MAXINT;
+      long min_delay = LONG_MAX;
 
       gettimeofday( &now, NULL );
       for ( int j = 0; j < n_monitors; j++ )
